Resources/Locale/en-US/markings/harpy.ftl

```diff
@@ -1,8 +1,20 @@
-marking-HarpyWingDefault = Basic Wings
-marking-HarpyWingDefault-harpy = Wings
+marking-HarpyWingDefaultHuescale = Basic Wings (Huescale)
+marking-HarpyWingDefaultHuescale-huescaleharpy = Wings
 
-marking-HarpyWingFolded = Folded Wings
-marking-HarpyWingFolded-harpyfolded = Wings
+marking-HarpyWingDefaultWhitescale = Basic Wings (Whitescale)
+marking-HarpyWingDefaultWhitescale-whitescaleharpy = Wings
+
+marking-HarpyWingFoldedHuescale = Folded Wings (Huescale)
+marking-HarpyWingFoldedHuescale-huescaleharpyfolded = Wings
+
+marking-HarpyWingFoldedWhitescale = Folded Wings (Whitescale)
+marking-HarpyWingFoldedWhitescale-whitescaleharpyfolded = Wings
```

Comment:
```suggestion
marking-HarpyWingDefaultWhitescale = Basic Wings (Whitescale)
marking-HarpyWingDefaultWhitescale-whitescale_harpy = Wings

marking-HarpyWingFoldedHuescale = Folded Wings (Huescale)
marking-HarpyWingFoldedHuescale-huescale_harpy_folded = Wings

marking-HarpyWingFoldedWhitescale = Folded Wings (Whitescale)
marking-HarpyWingFoldedWhitescale-whitescale_harpy_folded = Wings
```

Why don't these and the one above not have underscores like the rest?

Reply:
They have to match the sprite states in the folder, would you like me to change those?

Reply:
It would be nice for consistency, yes.

Reply:
Done, my wrist hurts now.

Reply:
What? Okay..